sensor_array/analysis/sample_analysis.py

```diff
@@ -19,145 +20,22 @@ def yaml_loader(filepath):
     return data
 
 
-def import_simulated_data(sim_results, sort_by_gas=False, gas_for_sorting=None):
-    with open(sim_results) as file:
-        reader = csv.DictReader(file, delimiter='\t')
-        reader_list = list(reader)
-        keys = reader.fieldnames
+def load_filter_unite_henrys_data(filepath, gases, min_comp=0.0):
+    # ddf = dictionary of dataframes
+    ddf = {gas: pd.read_csv("%s%s.csv" %(filepath, gas), sep='\t', engine='python') for gas in gases}
+    ddf = {gas: ddf[gas][(np.isnan(ddf[gas].kh) == False) & (ddf[gas].max_comp >= 0.05)] for gas in gases}
+    all_mofs = [set(ddf[gas]['MOF']) for gas in gases]
+    common_mofs = all_mofs[0].intersection(*all_mofs[1::])
+    ddf = {gas: ddf[gas][ddf[gas].MOF.isin(common_mofs)] for gas in gases}
+    df_unified = pd.DataFrame(common_mofs, columns=['MOF'])
```

Comment:
This looks like the same function as the previous PR. Is this a duplicate?

Reply:
It is a duplicate (I thought I had that in the commit message, but maybe deleted it during a rebase). I plan to move a single copy into a single file at some point in the near future.